src/NzbDrone.Core/Notifications/Trakt/Resource/TraktCollectMoviesResource.cs

```diff
@@ -6,4 +6,9 @@ public class TraktCollectMoviesResource
     {
         public List<TraktCollectMovie> Movies { get; set; }
     }
+
+    public class TraktMoviesResource
```

Comment:
Do we really need a separate Resource, the add resource won't also work? (assuming trakt api will just ignore the other properties)